flink-staging/flink-streaming/flink-streaming-core/src/main/java/org/apache/flink/streaming/api/functions/sink/WriteSinkFunction.java

```diff
@@ -56,7 +56,7 @@ protected void cleanFile(String path) {
 			writer.print("");
 			writer.close();
 		} catch (FileNotFoundException e) {
-			throw new RuntimeException("File not found " + path, e);
+			throw new RuntimeException("An error occurred while cleaning the file:" + e, e);
```

Comment:
Many thanks!
Just in another PR, you told me change the `LOG("value=" + value.toString())` to `LOG("value=" + value)`.
So maybe I had some misunderstanding before.
:-D